tests/qchem/test_observable_hf.py

```diff
@@ -19,6 +19,8 @@
 import pennylane as qml
 from pennylane import numpy as np
 from pennylane import qchem
+from pennylane.pauli import pauli_sentence
+from pennylane.operation import enable_new_opmath, disable_new_opmath
```

Comment:
Why there are `enable_new_opmath` and `active_new_opmath`? Can't just have one thing?